io-engine/src/bdev/nexus/nexus_child.rs

```diff
@@ -909,7 +937,9 @@ impl<'c> NexusChild<'c> {
         }
 
         match state {
-            ChildState::Open | ChildState::Faulted(Reason::OutOfSync) => {
+            ChildState::Open => {
+                // TODO: double-check interaction with rebuild job logic
```

Comment:
Seems ok as looks like only a the ChildState::Open children will be part of the I/O path